src/app/components/Layout/FormContentArea/FormContentArea.js

```diff
@@ -39,7 +39,7 @@ function FormContentArea(props) {
   }, [])
 
   return (
-    <Box display="flex" px={45} py={20} my={20} overflow="auto">
+    <Box display="flex" px={45} my={3} py={3} overflow="auto" height="100%">
```

Comment:
`px={45}` wrong parameter. Qonsoll box takes coefficient, not direct quantity of "px". If I'm correct, max coefficient is 8.

Reply:
but it still work, I need between 5 and 4 coefficient, and got it this way